CONTRIBUTING.md

```diff
@@ -81,14 +81,14 @@ rules on your computer. We will now walk you through the steps to get going:
 
     ```bash
     # Linux
-    sudo ip route add 172.17.0.0/16 via $(minikube ip)
+    sudo ip route add $(kubectl get node minikube -o jsonpath="{.spec.podCIDR}") via $(minikube ip)
     # later on you can undo this with
-    sudo ip route del 172.17.0.0/16
+    sudo ip route del $(kubectl get node minikube -o jsonpath="{.spec.podCIDR}")
 
     # MACOS
-    sudo route -n add -net 172.17.0.0/16 $(minikube ip)
+    sudo route -n add -net $(kubectl get node minikube -o jsonpath="{.spec.podCIDR}") $(minikube ip)
     # later on you can undo this with
-    sudo route delete -net 172.17.0.0
+    sudo route delete -net $(kubectl get node minikube -o jsonpath="{.spec.podCIDR}")
```

Comment:
FWIW, I've never run any of these `route` commands on mac and everything has worked fine. I'm not sure what problem it solves, though? This line does get the right podCIDR.

Reply:
@minrk without these, the hub can't talk to the spawned pods. But I suspect that it's now only needed when the driver is qemu not docker.

Reply:
Yeah, I don't think I've ever done this, either when I used minikube in a VM in the olden days, or now with docker. I don't think I ever ran minikube with --driver=qemu, though.

Should it just be removed, or left in a "you may also need this if..." section? I don't know the situation on linux.

Reply:
Hmm, I definitely ran into errors with jupyterhub trying to reach out to the spawned pods without this. I didn't specify `--driver=qemu` - if you don't have docker desktop or another docker thing running, `--driver=qemu` is the default. 

I agree it could be moved into a subsection. I can dig up the failure mode when this isn't done.